finagle-thrift/src/main/ruby/README.md

```diff
@@ -0,0 +1,10 @@
+# Finagle - Trace
```

Comment:
I don't think this belongs in a README. This is already explained in the comment below. Remove this file altogether?

(A README file should give an idea of what this code is doing and how it should be used)